### PR TITLE
Remove support for openshift labels

### PR DESCRIPTION
**Pull request by mkosiarc**
This was replaced by tekton results

CLOUDBLD-9046

Signed-off-by: mkosiarc <mkosiarc@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- n/a Python type annotations added to new code
- n/a JSON/YAML configuration changes are updated in the relevant schema
- n/a Changes to metadata also update the documentation for the metadata
- n/a Pull request has a link to an osbs-docs PR for user documentation updates
- n/a New feature can be disabled from a configuration file

### CODE REVIEW

**Review by hjmodi — APPROVED** [review]

**Review by tkdchen — APPROVED** [review]

File: atomic_reactor/inner.py [hunk 1]
```diff
@@ -336,10 +336,8 @@ class ImageBuildWorkflowData(ISerializer):
     # post-build plugins (such as koji_import) to record them.
     all_yum_repourls: List[str] = field(default_factory=list)
 
-    # Plugins can store info here using the @annotation, @annotation_map,
-    # @label and @label_map decorators from atomic_reactor.metadata
+    # Plugins can store info here using the @annotation, @annotation_map
```
Comment (chmeliik) [inline atomic_reactor/inner.py:339]: The comment looks a bit incomplete (maybe just remove "the"?)

File: atomic_reactor/plugins/post_koji_import.py [hunk 2]
```diff
@@ -83,7 +82,6 @@
 ]
 
 
-@label('koji-build-id')
```
Comment (chmeliik) [inline atomic_reactor/plugins/post_koji_import.py:86]: We need this one (could be in annotations instead, maybe)
https://github.com/containerbuildsystem/atomic-reactor/blob/c0659607cebd90ead55e53591b6c027e3b6fdd34/tekton/tasks/source-container-set-results.yaml#L32
https://github.com/containerbuildsystem/atomic-reactor/blob/c0659607cebd90ead55e53591b6c027e3b6fdd34/tekton/tasks/binary-container-set-results.yaml#L34

Comment (rcerven) [reply atomic_reactor/plugins/post_koji_import.py:86]: so change it to annotations and change it also in those tasks as that is required

**Review by rcerven — CHANGES_REQUESTED** [review]

File: atomic_reactor/inner.py (hunk 1 above)
Comment (mkosiarc) [reply atomic_reactor/inner.py]: updated

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

**Review by chmeliik — APPROVED** [review]

**Pull request merged**